### ISSUE TITLE
Thor_Proc_GetISXOptions is not found

### ISSUE THREAD

**Issue by TheFellow** [VFPX/PEMEditor#10]
I've done a clean installation of Thor, installed the PEMEditor and everything from the repository.
After adding a property to a class I receive an error that oISXOptions is not an object.

I tried stepping into it and it's failing on the line `Execscript (_Screen.cThorDispatcher, 'Thor_Proc_GetISXOptions')`

I've looked everywhere I can think of to find this thor proc prg but am at a loss.
Since the defintion of the isx form appears to be in this project I expected to find it here, but alas...

Am I missing something? Any direction would be helpful.

**Comment by TheFellow** [comment]
Thanks @DougHennig. I did the manual installation indicated on the Thor Repository. I had an issue where checking for updates resulted in a blocked page (I'm behind a firewall at work) so I just downloaded the zip that's there and unzipped it into the Tools directory as indicated. The GetISXOptions proc does not appear to be included in the zip file.

It looks like this should be associated with the IntellisenseX project, but I don't have all of the IntellisenseX menu entries shown in some of the screenshots from the Thor readme
![image](https://user-images.githubusercontent.com/2533155/129400140-bc4ce508-1b51-4214-ad83-ea5527abda04.png)
